docker/dyno/app/__init__.py

```diff
@@ -20,6 +20,11 @@ def create_app(config_class=Cfg):
 app = create_app()
 
 
+@app.after_request
```

Comment:
This is super stupid change, suggested by lighthouse to cache static resources in the browser

Reply:
I don't understand why we would add this. We're already sending back HTTP 304s and Cache-Control headers are already being set correctly.
<img width="1792" alt="Capture d’écran 2021-01-20 à 10 44 45" src="https://user-images.githubusercontent.com/111616/105156920-5c366e80-5b04-11eb-866e-9493b8052318.png">
<img width="545" alt="Capture d’écran 2021-01-20 à 10 45 47" src="https://user-images.githubusercontent.com/111616/105156923-5ccf0500-5b04-11eb-8971-58fc3e5f909f.png">

Reply:
I'm ok with reverting it :)